libraries/AP_Arming/AP_Arming.h

```diff
@@ -36,6 +36,7 @@ class AP_Arming {
         ARMING_CHECK_CAMERA      = (1U << 16),
         ARMING_CHECK_AUX_AUTH    = (1U << 17),
         ARMING_CHECK_VISION      = (1U << 18),
+        ARMING_CHECK_DISARM      = (1U << 19),
```

Comment:
We probably don't want to set aside an arming bit for this.  Is there an existing bit which looks appropriate?

We could also consider the implications of just putting it behind arming-check-all.

Reply:
I couldn't find a suitable one.
But, I fixed about ARMING_CHECK_ALL.